tests/unit/test_providers.py

```diff
@@ -94,14 +109,14 @@ def test_complete_provider_process(provider_class, raw_file, results_file):
     # If the Provider test is using the GENERIC_ICAL_DATA_PATH it comes with a well-defined 'organizer'
     # from the notificaction.
     if provider_class == GenericProvider or raw_file == GENERIC_ICAL_DATA_PATH:
-        expected_result["organizer"] = "mailto:noone@example.com"
+        expected_result[0]["organizer"] = "mailto:noone@example.com"
```

Comment:
@carbonarok why not just put this check into a `for loop`?

Reply:
ah yeah good idea